Networking/Networking/Remote/BlazeRemote.swift

```diff
@@ -143,4 +175,65 @@ private extension BlazeRemote {
         static let postDate = "post_date"
         static let desc = "desc"
     }
+
+    enum Constants {
+        static let dateFormat = "yyyy-MM-dd"
+    }
+}
+
+/// Blaze Forecasted Impressions input
+public struct BlazeForecastedImpressionsInput: Encodable {
+    // Start date of the campaign.
+    let startDate: Date
+    // End date of the campaign
+    let endDate: Date
+    // Formatted string of the total budget of the campaign
+    let formattedTotalBudget: String
```

Comment:
This is still an ongoing conversation whether we want to use string or number, and whether it should be total budget or budget cents.

Since I'm not sure when this will get an update, let's keep it like this for now and update in a separate PR.